components/client/src/main/java/com/hotels/styx/client/StyxHttpClient.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2013-2018 Expedia Inc.
- *
+ * <p>
```

Comment:
Get rid of these `<p>`s - they cause the copyright notice to get added again when you build.

Reply:
OMG why do they keep appearing all the time? Must be an IntelliJ Idea setting somewhere.